src/main/java/seedu/address/ui/IterationListItem.java

```diff
@@ -41,7 +43,9 @@ public IterationListItem(Iteration iteration, int displayIndex) {
         date.setText(iteration.getDate().date.toString());
         description.setText(iteration.getDescription().description);
         feedback.setText(iteration.getFeedback().feedback);
-        artImage.setImage(placeholder);
+        String fxImagePath = iteration.getImagePath().path;
+        artImage.setImage(new Image(new File(System.getProperty("user.dir")
```

Comment:
There were some issues with `user.dir` in the forum before (Issue 199) - I am not sure if this will affect us.